ddtrace/contrib/pytest/__init__.py

```diff
@@ -23,8 +23,8 @@
    The ddtrace plugin for pytest has the side effect of importing the ddtrace
    package and starting a global tracer.
 
-   If this is causing issues for your pytest runs where traced execution of
-   tests is not enabled, you can deactivate the plugin::
+   While you can avoid this by setting ``DD_TRACE_ENABLED=False``, if this is still causing issues for your pytest
```

Comment:
```suggestion
   Generating traces can be avoided by setting ``DD_TRACE_ENABLED=False``. If other issues occur then the plugin can be disabled entirely::
```

setting DD_TRACE_ENABLED does not avoid the import of ddtrace and starting a global tracer